subprojects/packagefiles/taglib/meson.build

```diff
@@ -0,0 +1,310 @@
+project('taglib', 'cpp',
+  version : '1.12',
```

Comment:
You are using one feature that requires meson 0.46.0

Because I am a nitpicker (sorry) I actually think it would be neat if we can use `meson_version:` here to declare that.

Reply:
I think this should do it.